### ISSUE TITLE
Array iteration + comment break

### ISSUE THREAD

**Issue by bastelfreak** [kuleuven/puppet-lint-manifest_whitespace-check#5]
Hi, it's me again! (sorry)

given the following code:

```puppet
class profiles::api {
  # create some firewallrules
  # this profile is for a management node, it needs http/https access to other services
  ['api', 'api.staging', 'staging.api'].each |$node| {
    @@ferm::rule { "allow_http_https_${trusted['certname']}_${node}":
      chain  => 'INPUT',
      action => 'ACCEPT',
      proto  => 'tcp',
      dport  => '(80 443)',
      saddr  => $facts['networking']['ip'],
      tag    => $node,
    }
  }
}
```

results into this diff (tested with 0.1.9):

```diff
diff --git a/manifests/api.pp b/manifests/api.pp
index b0496331..ba320d14 100644
--- a/manifests/api.pp
+++ b/manifests/api.pp
@@ -1,14 +1,13 @@
 class profiles::api {
   # create some firewallrules
-  # this profile is for a management node, it needs http/https access to other services
-  ['api', 'api.staging', 'staging.api'].each |$node| {
-    @@ferm::rule { "allow_http_https_${trusted['certname']}_${node}":
-      chain  => 'INPUT',
-      action => 'ACCEPT',
-      proto  => 'tcp',
-      dport  => '(80 443)',
-      saddr  => $facts['networking']['ip'],
-      tag    => $node,
-    }
+  # this profile is for a management node, it needs http/https access to other services ['api', 'api.staging', 'staging.api'].each |$node| {
+  @@ferm::rule { "allow_http_https_${trusted['certname']}_${node}":
+    chain  => 'INPUT',
+    action => 'ACCEPT',
+    proto  => 'tcp',
+    dport  => '(80 443)',
+    saddr  => $facts['networking']['ip'],
+    tag    => $node,
   }
 }
+}
```

the plugin somehow swollows the `\n` :(

**Comment by jovandeginste** [comment]
Should be fixed in 0.1.16

**Comment by bastelfreak** [comment]
can confirm it's fixed \o/
